src/reducers/RootReducer.js

```diff
@@ -9,8 +9,6 @@ import bales from './BalesReducer';
 import createBaleModal from './CreateBaleModalReducer';
 import editBaleModal from './EditBaleModalReducer';
 import createPocketModal from './CreatePocketModalReducer';
-import date from './GatherReducer';
-import hour from './GatherReducer';
 import travelImage from './GatherReducer';
 import kmsTraveled from './GatherReducer';
 import pocketsCollected from './GatherReducer';
```

Comment:
You should write the import attributes in the same line like the further example: { travelImage, pocketsCollected, ... } from './GatherReducer';  instead of one import at a time.

Reply:
I don't know why but when importing everything in a single line like { travelImage, kmsTraveled, pockesCollected } the compiler doesn't find the attributes.